packages/documentation/src/pages/getting-started/common-tasks/new-page.mdx

```diff
@@ -0,0 +1,16 @@
+---
+title: Creating a new content page
+tags: content, vagov-content, markdown, frontmatter
+---
+
+# Creating a new content page
+
+Note: Check out the [Adding a new feature](/getting-started/common-tasks/new-feature) doc to make sure you're trying to add the right type of feature to VA.gov.
+
+## Choosing vagov-content or Drupal
+
+There are two ways to add a new page to VA.gov; one is by adding it in vagov-content. The other is to add it to the Drupal CMS. Drupal will eventually be the only source of content, but `vagov-content` is expected to be around for the near future.
+
+If you need to add a page to `vagov-content`, there is [documentation](https://github.com/department-of-veterans-affairs/vagov-content/blob/master/docs/index.md) in that repo. You should also contact the website team to coordinate with them on where the new page should live.
+
+If you need to add a page to Drupal, coordinate with the cms team.
```

Comment:
might be nice to have some info re: how to contact the CMS team

Reply:
Yeah, I'm not sure what to add for that. Generally, this will be figured out on your team and you won't be looking in the docs for this info (hopefully). I'll add something about going through a PM.